tests/Jellyfin.Plugin.Bookshelf.Tests/ComicBookInfoProviderTest.cs

```diff
@@ -0,0 +1,250 @@
+using System;
+using Jellyfin.Plugin.Bookshelf.Providers.ComicBookInfo;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.IO;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+
+namespace Jellyfin.Plugin.Bookshelf.Tests;
+
+public class ComicBookInfoProviderTest
+{
+    private readonly ComicBookInfoFormat _comicBookInfoFormat;
+
+    private readonly ComicBookInfoProvider _uut;
+
+    public ComicBookInfoProviderTest()
+    {
+        _comicBookInfoFormat = GenerateTestData();
+
+        var fileSystem = Substitute.For<IFileSystem>();
+        var logger = Substitute.For<ILogger<ComicBookInfoProvider>>();
+        _uut = new ComicBookInfoProvider(fileSystem, logger);
```

Comment:
What does uut stand for? I'm familiar with `System Under Test` (SUT), but not uut

Reply:
`Unit Under Test`
Not sure which one is used more often, but I'm familiar with both terms.